docs/reference/supervisor.md

```diff
@@ -1708,7 +1708,7 @@ void wb_supervisor_node_add_torque(node, torque, relative)
 
 *add force or torque to a Solid node.*
 
-The `wb_supervisor_node_add_force` function adds a force to the [Solid](solid.md) node at its center of mass, the `relative` argument defines if the force is expressed in world coordinate system or relatively to the node.
+The `wb_supervisor_node_add_force` function adds a force to the [Solid](solid.md) node at its center of mass, the `relative` argument defines if the force is expressed in world coordinate system (`relative = False`) or relatively to the node (`relative = True`).
```

Comment:
Because in C/C++, we use `true` and `false` instead of `True` and `False` (in Python), I would rather formulate this in a more generic way:

```suggestion
The `wb_supervisor_node_add_force` function adds a force to the [Solid](solid.md) node at its center of mass, the `relative` argument defines if the force is expressed in world coordinate system (`relative` set to false) or relatively to the node (`relative` set to true).
```